### DISCUSSION TITLE
Feature: Range Slider

### DISCUSSION THREAD

**Discussion by Silthus** [tamagui/tamagui#1222] [Ideas]
It would be great if the slider could be extended to work as a range slider or another component  is created allow to selecting a low and high value.

**Comment by natew** [comment]
It does have this, but we need to document it for sure and make sure it's tested.

**Reply by ryanchapel-hero** [reply]
Any info on how this range slider functionality works?

**Comment by Silthus** [comment]
> It does have this, but we need to document it for sure and make sure it's tested.

Nice, but how does it work? I looked through the source of the `Slide` and couldn't figure out how to get two thumb knobs on it that each select their own value (low and high).

Can you point me in the right direction?
